app/routes/account.py

```diff
@@ -32,9 +32,11 @@ def create_token():
         uns = UserNotificationService(user)
 
         if token_type == "email_confirm":
-            uns.activation_emails()
+            if not user.confirmed:
```

Comment:
Isn't this already handled in `UserNotificationService`? `user_notification_service.py` line 18 and 27